### PR TITLE
Hide features row if tx has no features

### LINKED ISSUE #3323: Hide feature row when there are no features
Screenshot 2023-03-12 at 17 20 51

### PR DESCRIPTION
**Pull request by mononaut**
Fixes #3323 

| Before | After |
|-|-|
| Screenshot 2023-03-13 at 12 50 29 PM | Screenshot 2023-03-13 at 12 50 18 PM |

### CODE REVIEW

File: frontend/src/app/components/transaction/transaction.component.ts [hunk 1]
```diff
@@ -495,6 +517,7 @@ export class TransactionComponent implements OnInit, AfterViewInit, OnDestroy {
 
   @HostListener('window:resize', ['$event'])
   setGraphSize(): void {
+    this.isMobile = window.innerWidth < 850;
```
Comment (nymkappa) [inline frontend/src/app/components/transaction/transaction.component.ts:520]: Usually better to use media query for showing/hiding html elements

File: frontend/src/app/components/transaction/transaction.component.ts [hunk 2]
```diff
@@ -428,9 +436,23 @@ export class TransactionComponent implements OnInit, AfterViewInit, OnDestroy {
       });
   }
 
+  setFeatures(): void {
+    if (this.tx) {
+      this.segwitEnabled = !this.tx.status.confirmed || this.tx.status.block_height >= 477120;
```
Comment (nymkappa) [inline frontend/src/app/components/transaction/transaction.component.ts:441]: Isn't this going to break on testnet/signet?

File: frontend/src/app/components/transaction/transaction.component.ts (hunk 1 above)
Comment (mononaut) [reply frontend/src/app/components/transaction/transaction.component.ts:520]: We need to add/remove the actual table row markup element to get the correct alternating table stripes.

`display: none` via CSS media query isn't enough.

**Review by softsimon — APPROVED** [review]
tested ACK @ v2.5.0-dev [[130ae8c3a](https://github.com/mempool/mempool/commit/130ae8c3a)]

**Pull request merged**